addons/definition.json

```diff
@@ -0,0 +1,4 @@
+{
+  "game": "game/dota_addons/{ADDON_NAME}/",
+  "content": "content/dota_addons/{ADDON_NAME}/"
+}
```

Comment:
Whoops, that one was caused by a mistake I made when setting up node.js. I should be good to just delete it, right?